contents/blog/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,134 @@
+
+PostHog works like a group of startups with lots of [small teams](/handbook/company/small-teams). Each of our products has a team of up to 6 who can ship with minimal interference from the rest of the company. We hire a lot of former technical founders to help here.
+
+## Nearly everyone is an engineer
+
+It's simple, but if you have lots of spending and focus outside of engineering, you won't get as much built.
+
+At the time of writing, [we are 37 people](../team), only 11 of whom are in Go To Market or Operations roles (and 6 of those 11 have at least some engineering in their background).
+
+We don't do outbound sales, our [marketing team](/handbook/small-teams/marketing) is very small, we only have one product manager, our [exec team](/handbook/small-teams/exec) only has 3 people. These things are all by design (and were possible because of our market - the majority of engineers generally want docs and to try something instead of demos and salespeople to deal with).
+
+## No meeting Tuesdays/Thursdays
+
+We want a culture where people can deep work. We push back on unnecessary meetings (and yes, there are _definitely_ some that are necessary, we have learned - often painfully).
+
+## We are a late mover
+
+I've had lots of founders tell me that they can't pursue idea X because some other startups are tackling it. This makes me laugh because we simply won't build a new product unless there is a $1bn+ competitor already offering it. We _love_ competition. It shows you that there will be demand. The magic in our platform (after an individual product is working well for our users) is then integrating everything together.
+
+For non-God tier product people, such as me, I think it's easier to build a better version of X than it is to create something entirely new.
+
+## We raised venture capital
+
+Obviously, this matters. We didn't raise or spend a ridiculous amount compared to other companies we compete with – we have done a $3M seed, $9M series A and a $15M series B. We have a lot of it left and will be profitable next year.
+
+Raising money does let build more stuff, but if you _should_ is very questionable and [a topic for another time](vc-or-bootstrap).
+
+## Trust and feedback over process
+
+This is one of [our values](/handbook/company/values). It's simply up to the person building in most situations. Building and scaling something people want is a nuanced problem, so we let people use their judgement. When they get it wrong, we are direct and give feedback.
+
+To quote one of our team "process is scar tissue" – it often overcorrects, but because [humans are risk adverse](https://www.adamjuliangoldstein.com/blog/anxiety-algorithm/) it feels natural to implement them when a company starts growing. Humans, therefore, will overcorrect by default and that's why large corporations are (in more cases than not) irrationally-obstructive to getting work done.
```

Comment:
```suggestion
To quote one of our team: "process is scar tissue" – it often overcorrects, but because [humans are risk adverse](https://www.adamjuliangoldstein.com/blog/anxiety-algorithm/) it feels natural to implement it when a company starts growing. Humans, therefore, will overcorrect by default and that's why large corporations are (in more cases than not) irrationally-obstructive to getting work done.
```